trailblazer/dto/analyses_response.py

```diff
@@ -0,0 +1,52 @@
+from pydantic import BaseModel
+from datetime import datetime
+
+
+class Job(BaseModel):
+    analysis_id: int
+    context: str | None = None
```

Comment:
What context is this?

Reply:
No idea, it is a field on the database model for a job. Looking at the database, the values in the column seem to be case names.

Reply:
It is not used in the analyses table, so I removed it from the response